utils/torch_helper.py

```diff
@@ -49,3 +58,44 @@ def test_step(self, batch, batch_idx):
 
     def training_step(self, batch, batch_idx):
         return self.loss_logits_y(batch)[0]
+
+
+class DDPStrategyNoTeardown(DDPStrategy):
+    # from
+    # https://pytorch-lightning.readthedocs.io/en/1.6.0/_modules/pytorch_lightning/strategies/ddp.html#DDPStrategy.teardown
+    def teardown(self):
+        log.detail(f"{self.__class__.__name__}: tearing down strategy")
+        super().teardown()
```

Comment:
As you call this `teardown` I think the model i off-loaded no?

Reply:
Good call. Can we do super of the grand parent class ?

Reply:
I tried the solution from here: https://stackoverflow.com/a/48317309/4332585